Add integration page for homeassistant_sky_connect (#28739)
home-assistant/home-assistant.github.io: source/_integrations/homeassistant_sky_connect.markdown

---

> title: Home Assistant Sky Connect

Reviewer: ```suggestion
title: Home Assistant SkyConnect
```

> description: Home Assistant Sky Connect provides hardware information for the hardware configuration page.

Reviewer: ```suggestion
description: Home Assistant SkyConnect provides hardware information for the hardware configuration page.
```

ha_release: 2022.9
ha_category:
  - Other
ha_codeowners:
  - '@home-assistant/core'
ha_domain: homeassistant_sky_connect
ha_integration_type: hardware
---

> The Home Assistant Sky Connect integration provides hardware information for the hardware configuration page.

Reviewer: ```suggestion
The Home Assistant SkyConnect integration provides hardware information for the hardware configuration page.
```

> For documentation about the Home Assistant Sky Connect please visit the [documentation](https://skyconnect.home-assistant.io/documentation/).

Reviewer: ```suggestion
For documentation on the Home Assistant SkyConnect, please visit the [documentation page](https://skyconnect.home-assistant.io/documentation/).
If you are looking to buy one, please visit the [product page](https://home-assistant.io/skyconnect

---
title: Home Assistant SkyConnect
description: Home Assistant SkyConnect provides hardware information for the hardware configuration page.
ha_release: 2022.9
ha_category:
  - Other
ha_codeowners:
  - '@home-assistant/core'
ha_domain: homeassistant_sky_connect
ha_integration_type: hardware
---

The Home Assistant SkyConnect integration provides hardware information for the hardware configuration page.

For documentation on the Home Assistant SkyConnect, please visit the [documentation page](https://skyconnect.home-assistant.io/documentation/).

> If you are looking to buy one, please visit the [product page](https://home-assistant.io/skyconnect

Reviewer: ```suggestion
If you are looking to buy one, please visit the [product page](https://home-assistant.io/skyconnect)
```
